packages/nodes-base/nodes/Jira/GenericFunctions.ts

```diff
@@ -76,16 +76,23 @@ export async function jiraSoftwareCloudApiRequest(
 }
 
 export function handlePagination(
+	method: IHttpRequestMethods,
 	body: any,
 	query: IDataObject,
 	paginationType: 'offset' | 'token',
 	responseData?: any,
 ): boolean {
 	if (!responseData) {
 		if (paginationType === 'offset') {
-			query.startAt = 0;
-			query.maxResults = 100;
+			if (method === 'GET') {
+				query.startAt = 0;
+				query.maxResults = 100;
+			} else {
+				body.startAt = 0;
+				body.maxResults = 100;
+			}
 		} else {
+			// token pagination is used only in POST requests
 			body.maxResults = 100;
 		}
```

Comment:
@RomanDavydchuk could you please add comment with link to API documentation page covering this?